runtime/common/src/paras_registrar.rs

```diff
@@ -38,6 +38,9 @@ use runtime_parachains::{
 	Origin,
 };
 
+/// The sequence of bytes a valid wasm module binary always starts with.
+const WASM_MAGIC: &[u8] = &[0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00];
```

Comment:
can we extract it to lib or make it public so it can be reused?

Reply:
Sure, I could create a module which would host this exact const. That was my initial thinking as well of course.

But the question is, would that be beneficial?

Reply:
> Sure, I could create a module which would host this exact const.

Wait, why do you need a separate module for that?
I could put it in `runtime/common/src/lib.rs` as a pub const.
And then use it here and in `runtime/common/src/paras_sudo_wrapper.rs`

Reply:
Ah, ok, I prefer to not pollute lib.rs making it a clean entrypoint to the crate, but yeah, it already has a bunch of trash there so I guess one little constant won't hurt. 

I'll make the change, but TBH not sure if I agree that it's a win.